catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceInputTransformer.java

```diff
@@ -0,0 +1,333 @@
+            if (fileSize != null) {
+                metacard.setAttribute(Metacard.RESOURCE_SIZE, fileSize);
+            }
+        }
+        ArrayList<String> associations = new ArrayList<>();
+        if (StringUtils.isNotEmpty(baseUrl)) {
+            associations.add(String.format("%s%s", baseUrl, getString(links, "webui")));
+            associations.add(baseUrl);
+        }
+
+        String space = getStringOrDefault(json, null, "space", "_links", "webui");
+        if (space != null) {
+            associations.add(baseUrl + space);
+        }
+        metacard.setAttribute(Associations.EXTERNAL, associations);
+
+    }
+
+    private void parseBody(MetacardImpl metacard, Object json, String confluenceType) {
+        Object body = getJsonElement(json, "body", "view", "value");
+        if (body != null) {
+            String cleanedText = body.toString()
+                    .replaceAll("<.*?>", " ");
+            String description = cleanedText;
+            if (description.length() > DESCRIPTION_SIZE) {
+                description = description.substring(0, DESCRIPTION_SIZE) + "...";
+            }
+            if (StringUtils.isNotEmpty(description)) {
+                String xmlSafeString = StringEscapeUtils.escapeXml10(cleanedText);
+                metacard.setAttribute(Metacard.DESCRIPTION, description);
+                metacard.setAttribute(Metacard.METADATA,
+                        String.format(PRODUCT_XML, confluenceType, xmlSafeString));
+                metacard.setAttribute(Confluence.BODY_TEXT, xmlSafeString);
+            }
+        }
+        if (confluenceType.equals("attachment")) {
+            Object comment = getJsonElement(json, "metadata", "comment");
+            if (comment != null) {
+                metacard.setAttribute(Metacard.DESCRIPTION, comment.toString());
+            }
+            Object mediaType = getJsonElement(json, "metadata", "mediaType");
+            if (mediaType != null) {
+                metacard.setAttribute(Media.TYPE, mediaType.toString());
+            }
+        } else {
+            metacard.setAttribute(Media.TYPE, "text/html");
+        }
+    }
+
+    private void parseRestrictions(MetacardImpl metacard, Object json)
+            throws CatalogTransformerException {
+        Object restrictions = getJsonElement(json, "restrictions", "read", "restrictions");
+        if (restrictions != null) {
+            ArrayList<String> userRestrictions = new ArrayList<>();
+            ArrayList<String> groupRestrictions = new ArrayList<>();
+
+            getJsonArray(restrictions, "user", "results").stream()
+                    .forEach(e -> userRestrictions.add(getStringOrDefault(e, UNKNOWN, "username")));
+            getJsonArray(restrictions, "group", "results").stream()
+                    .forEach(e -> groupRestrictions.add(getStringOrDefault(e, UNKNOWN, "name")));
+            metacard.setAttribute(Security.ACCESS_INDIVIDUALS, userRestrictions);
```

Comment:
Will these individuals be email addresses and groups be attribute store roles/groups?  Or are you just trying to capture the information even if our policy extensions cannot match these values if they are not emails/roles?